edx_proctoring/api.py

```diff
@@ -205,6 +205,19 @@ def get_review_policy_by_exam_id(exam_id):
     return ProctoredExamReviewPolicySerializer(exam_review_policy).data
 
 
+def _get_review_policy_by_exam_id(exam_id):
+    """
+    Looks up exam by the primary key. Returns None if not found
+
+    Returns review_policy field of the Django ORM object
+    """
+    try:
+        exam_review_policy = get_review_policy_by_exam_id(exam_id)
+        return ProctoredExamReviewPolicySerializer(exam_review_policy).data['review_policy']
```

Comment:
Looks like the codecov failure may be a real one - this line doesn't looks to be hit by any tests: https://codecov.io/gh/edx/edx-proctoring/compare/fff5ab971d2de3491ac062ecef393a9afef613d8...6d8056c2c1311ca6e9e5a12c3f2f30521c5efda4/diff

Would it be possible to add/modify a test that does hit this return?